javascript/javascript1/week4/class_exercises/tic_tac_toe.js

```diff
@@ -0,0 +1,80 @@
+// still getting used to destructuring, .map() higher-order function and ternary operators!
+
+const position = [
+    ['x', 'o', ' '],
+    [' ', 'o', ' '],
+    [' ', 'o', 'x']
+    ];
+
+    function getRenderedGame(currentGameFlat) {
+        const separator = "*";
+
+        const displayGame = currentGameFlat.map(ticTacToeRow => {
+            const destructuredTicTacToeRow = [...ticTacToeRow];
+            return `*${destructuredTicTacToeRow.join(separator)}*`;
```

Comment:
This could be together 

`    const displayGame = currentGameFlat.map(ticTacToeRow => `*${ticTacToeRow.join(separator)}*`).join("\n");
`